initial_lat: 0.0
initial_lon: 0.0
dest_lat: 2.0
dest_lon: 2.0